proto/service/gribi.proto

```diff
@@ -14,10 +14,12 @@
 
 syntax = "proto3";
 
-import "github.com/openconfig/gribi/proto/gribi_aft/gribi_aft.proto";
-
 package gribi;
 
+import "proto/gribi_aft/gribi_aft.proto";
```

Comment:
similar to the previous comment in gribi_aft.proto. Just confirming the change to relative paths is intentional.